src/root-config.ts

```diff
@@ -0,0 +1,14 @@
+import { registerApplication, start } from "single-spa";
+import { registerAllCoreApplications } from "joeldenning-root-config/dist/root-config-lib";
```

Comment:
This is an example of how people will be able to configure the openmrs-esm-root-config:

1) npm install it.
2) import the lib.
3) call functions

We might need to revisit this with a future RFC, but this is what I'm thinking distributions will do to configure their microfrontends.

Reply:
I published the openmrs-esm-root-config as joeldenning-root-config because I don't have permissions to publish to the `@openmrs` organization on npm. See related https://talk.openmrs.org/t/adding-members-to-the-openmrs-npm-organization/23260/9